packages/primero/src/Adaptor.js

```diff
@@ -222,7 +235,11 @@ export function getCases(query, options, callback) {
           );
 
           if (expandedOptions?.withReferrals) {
+            console.log(
+              `This might take a while, we are fetching ${cases.length} cases referrals`
+            );
             for await (const c of cases) {
+              console.log(`Fetching ${c.id.slice(0, -10) + '...'}, referrals`);
```

Comment:
Well we can do this, sure, but I didn't ask for it.

it'd be quite happy to literally just `console.log('Fetching referral...')`. 

If you really want to help the user, try something like `fetching referral 1 of 27...`. Then you don't need the top line at all

Reply:
Okay let's use `Fetching referral...`. 
Although this is not useful when debugging. I wish i have more control of the logs when fetching referrals. Because when an error happen it's a bit painful figuring out what went wrong. 

But again the new CLI has better error message so maybe it will be easier